src/NodeServices/WorkspaceEvents.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace DSNodeServices
```

Comment:
That's a good point Ben. I had my blinders on about the user story having to do with accessing this from node libraries, but Dynamo.Services make more sense.